packages/store/src/storagecoder/StorageCoder.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { Slice, Slice_ } from "../Slice.sol";
+import { SchemaType } from "../Types.sol";
+
+library StorageCoder {
```

Comment:
Naming is hard; but technically this library just does our custom tight packing / encoding / unpacking / decoding, it's not strictly connected to storage. Don't really have a much better proposal either, will think about it more (planning to do a more in-depth review tomorrow)

Reply:
I kind of like `TightCoder`, makes the purpose pretty obvious. Also maybe @holic has an idea, he's great with naming.

Reply:
I don't love TightCoder haha

DataPack? DataPacker? BytePacker? BytesPacker? DataCoder? ByteCoder? 🤷‍♂️

Reply:
how about Serializer?

Reply:
> another thought: since this is only a few methods, would it make sense to include them inside another lib (e.g. SliceLib) we already have?

Well, there're only 2 core encode/decode methods, but then there're like 200+ for a plethora of solidity types that use these 2. I'd rather isolate them as much as possible, since it's a ton of code.
They do depend on Slice, but that could actually change, and Slice doesn't depend on them.